book/1 - Installing/2 - Compiling Cuberite Yourself.html

```diff
@@ -1,16 +1,16 @@
 <p>
 	Compiling yourself takes longer and is more involved, but on modern processors can lead to a speed increase of up to 1.5 to 3x.
-	If your operating system is not officially supported, compiling may be the only way to run Cuberite.
+	If your operating system or hardware is not officially supported, compiling may be the only way to run Cuberite.
 </p>
 
 <p>
 	The automatic compilation script is recommended.
-	It takes care of the compilation process for you. You only need to copy this command to your Bash terminal:
+	It takes care of the compilation process for you. You only need to copy this command to your terminal:
```

Comment:
Sorry, my bad.

It works on all `sh`-supporting systems. But it won't work on Windows.